thenewboston_node/business_logic/tests/test_models/test_field_compaction.py

```diff
@@ -103,3 +103,102 @@ def test_can_load_block_from_compacted_dict():
     loaded_block = Block.from_compact_dict(compacted)
 
     assert loaded_block == block
+
```

Comment:
add unittest the switching hexadecimals to binary result into reduced serialized size

Reply:
https://github.com/thenewboston-developers/thenewboston-node/pull/114/files#diff-638e0785bcd0039bf74e9df592e40ef1fa8573cfd0afe997ff0a1aebbfe073eeR222